winlogbeat/checkpoint/file_windows.go

```diff
@@ -29,8 +30,8 @@ func createWriteThroughFile(path string) (*os.File, error) {
 		syscall.GENERIC_READ|syscall.GENERIC_WRITE,               // Access Mode
 		uint32(syscall.FILE_SHARE_READ|syscall.FILE_SHARE_WRITE), // Share Mode
 		nil, // Security Attributes
-		syscall.CREATE_ALWAYS,                                          // Create Mode
-		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH), // Flags and Attributes
+		syscall.CREATE_ALWAYS, // Create Mode
+		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH|_FILE_FLAG_NO_BUFFERING), // Flags and Attributes
```

Comment:
This is what originally pointed me to FILE_FLAG_WRITE_THROUGH. https://stackoverflow.com/questions/6525943/cause-of-corrupted-file-contents/24302389#24302389